### DISCUSSION TITLE
episode/S01E16

### DISCUSSION THREAD

**Discussion by giscus** [opensource-f2f/website#43] [General]
# episode/S01E16

《开源面对面》是个系列访谈节目，旨在通过采访不同行业、岗位的开源参与者的亲身经历、心得体会，让更多的从业人员感受到参与开源是一种积极的趋势与良好的 职业发展方向，认识到任何人都可以参与开源，并在开源的世界里体现自己的价值。

https://osf2f.net/episode/S01E16

**Comment by xiaowangxiaowang256256** [comment]
1、我是一名初级前端开发者，了解《开源面对面》源于，一次前端开发中，input组件出现的bug。我通过查阅element-plus的[更新文档](https://element-plus.gitee.io/zh-CN/guide/changelog.html) ，尝试将版本升级至2.5.2后，解决了该bug。通过@YoungDan-hero的[代码贡献](https://github.com/element-plus/element-plus/pull/15514 )，解决了我的工作问题。通过回溯到上游（[element-plus](https://github.com/element-plus/element-plus)），借助上游社区的力量，来解决下游项目开发的问题。这真的是一件激动人心的事情。这是一种全新的开发体验，它给我的项目开发，提供了一个全新的视角。

2、由于经常了解element-plus开源项目  ，便关注到了@sxzz，阅读他的[个人博客](https://xlog.sxzz.moe/)  ，并了解了开源面对面的[节目采访音频](https://osf2f.net/episode/S01E16)  。之后在上下班通勤路上，便听起来了《[开源面对面](https://osf2f.net/)》的音频  。

3、收听开源音频，关注开源项目，它帮助我解决了一些工作上的问题，拓宽了我的技术视野，收益良多。如今使用开源，了解开源，已经成为了我的兴趣爱好。也希望后续不断参与到开源的贡献中来，来传承发扬这种开源精神。

**Reply by LinuxSuRen** [reply]
我的项目 [api-testing](https://github.com/LinuxSuRen/api-testing) 也用到了 element-plus，很好的一个项目。
